tests/__init__.py

```diff
@@ -0,0 +1,28 @@
+import os
+import shutil
+
+SUPERSET_TEST_HOME = os.path.join(os.path.dirname(__file__), '.superset')
```

Comment:
If we don't do this, our test superset data gets shoved in our home directory, which is kinda ugly/not repeatable.